modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -87,12 +87,16 @@ spec:
       name: minio-test
       type: s3
   storageSize: 1Gi
+  tenants:
+    mode: openshift 
+    authentication: 
+      - tenantName: dev 
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" 
   template:
     gateway:
-      enabled: false
+      enabled: true
     queryFrontend:
       jaegerQuery:
-        enabled: true
         monitorTab:
           enabled: true # <1>
           prometheusEndpoint: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091 # <2>
```

Comment:
Need to remove this from yaml as we are enabling gateway.
        ingress:
          type: route